torch/_dynamo/variables/builtin.py

```diff
@@ -1546,11 +1546,8 @@ def call_getattr(
                         f"pending mutation on nn module, so graph breaking at {name!r} call"
                     )
 
-            try:
-                # re-read a pending side effect?
-                return tx.output.side_effects.load_attr(obj, name)
-            except KeyError:
-                pass
+        is tx.output.side_effects.has_pending_mutation_of_attr(obj, name)
```

Comment:
if

Reply:
Haha, I forgot to rerun ghstack. I was surprised to see the number of faiures.